reference/storefront_tokens.v3.yml

```diff
@@ -184,8 +185,15 @@ components:
         - oneOf:
           - $ref: "#/components/schemas/Channels"
           - $ref: "#/components/schemas/Channel"
+        - type: object
+          properties:
+            dev_mode:
+              type: boolean
+              description: Determines the operations you can perform with the token when a storefront is in [prelaunch](https://support.bigcommerce.com/s/article/Multi-Storefront#configuring) or [maintenance](https://support.bigcommerce.com/s/article/Maintenance-Mode#enabling) (i.e. development mode). When set to true, you can perform all operations, including [login and logout mutations](/docs/start/authentication/graphql-storefront#signing-customers-in). When set to false, you cannot perform login and logout mutations.
+              default: false
       required:
         - expires_at
+        - dev_mode
```

Comment:
`dev_mode` isn't a required parameter, but looks like it's been added to required fields list.

Reply:
Nice catch, I fixed it